src/main/java/core/basesyntax/TriangleArea.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface TriangleArea {
```

Comment:
Don't use verbs for class/interface names. Interfaces should represent a capability or a characteristic, such as 'AreaCalculatable' or 'Shape', rather than an action. Consider renaming this interface to better reflect its intent.